File: Logica_de_Programacao_Python/Desafio_5/desafio_5/Desafio5.py
def calcular_comissao(valor_imovel):

    if valor_imovel >= 50000:
        comissao = valor_imovel * 0.20  # 20% de comissão
    elif valor_imovel >= 30000:
        comissao = valor_imovel * 0.15  # 15% de comissão
    else:
        comissao = valor_imovel * 0.10  # 10% de comissão
    return comissao

def main():
    print('-------------------------------')
    print("Bem-vindo ao sistema de comissão de vendas!")
    print('-------------------------------\n')

    nome_vendedor = input("Informe o nome do vendedor: ")
    valor_imovel = float(input("Informe o valor do imóvel a ser vendido (R$): "))

    comissao = calcular_comissao(valor_imovel)

    # Exibe as informações solicitadas
    print("\nResumo da venda:")
    print(f"Vendedor: {nome_vendedor}")
    print(f"Valor do imóvel: R$ {valor_imovel:.2f}")
    print(f"Comissão: R$ {comissao:.2f}")

if __name__ == "__main__":
    main()

File: Logica_De_Programacao_Java/Desafio_5/src/Desafio_5.java
import java.util.Scanner;

public class Desafio_5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o nome do vendedor: ");
        String nomeVendedor = scanner.nextLine();

        System.out.print("Digite o valor do imóvel: ");
        double valorImovel = scanner.nextDouble();

        double comissao = 0;
        double porcentagem = 0;

        if (valorImovel >= 50000) {
            comissao = valorImovel * 0.20;
            porcentagem = 20;
        } else if (valorImovel >= 30000) {
            comissao = valorImovel * 0.15;
            porcentagem = 15;
        } else {
            comissao = valorImovel * 0.10;
            porcentagem = 10;
        }

        System.out.println("Nome do vendedor: " + nomeVendedor);
        System.out.println("Valor do imóvel: R$ " + valorImovel);
        System.out.println("Comissão: R$ " + comissao + " (" + porcentagem + "%)");

        scanner.close();
    }
}